### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by odinuv**
feat: this feature has been moved to data-loader
keboola/data-loader#26

### CODE REVIEW

File: code/rsession_init.R [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
+        rscript = readChar('/data/main.R', file.info('/data/main.R')$size)
+    } else {
+        rscript = ''
     }
     fileName <- '/tmp-rstudio/templatefile.json'
     data <- readChar(fileName, file.info(fileName)$size)
     configData <- jsonlite::fromJSON(data)
     configData$contents <- rscript
     jsonData <- jsonlite::toJSON(configData, auto_unbox = TRUE, pretty = TRUE)
     writeChar(jsonData, paste0('/data/.rstudio/sdb/per/t/AAAAAAA'), eos = NULL)
```
Comment (pivnicek) [inline code/rsession_init.R]: @odinuv sorry for the drive-by comment, but I'm curious about this line:)

Comment (odinuv) [reply code/rsession_init.R]: if you have an open file in rstudio it's content is written both in the file and both in it's rstudio session json. if one of them is missing it says something like 'the file has changed on the disk, reload?' while this doesn't solve it completely (you can still sometimes see the message) it does not really matter whether you overwrite or not, because the contents are actually the same.
